apple/Handlers/RNNativeViewHandler.mm

```diff
@@ -39,20 +39,27 @@ - (id)initWithGestureHandler:(RNGestureHandler *)gestureHandler
 
 - (void)touchesBegan:(NSSet<RNGHUITouch *> *)touches withEvent:(UIEvent *)event
 {
+  [self setStateIfScrollView:UIGestureRecognizerStatePossible];
   [_gestureHandler setCurrentPointerType:event];
   [_gestureHandler.pointerTracker touchesBegan:touches withEvent:event];
 }
 
 - (void)touchesMoved:(NSSet<RNGHUITouch *> *)touches withEvent:(UIEvent *)event
 {
+  [self setStateIfScrollView:UIGestureRecognizerStateChanged];
```

Comment:
Now that I think about it, can we just copy the state of the scroll view's recognizer here?